djangocms_navigation/admin.py

```diff
@@ -269,6 +275,13 @@ def __new__(cls, *args, **kwargs):
 
         return Form
 
+    def get_object_url(self, obj):
+        if obj.content:
+            obj_url = obj.content.get_absolute_url()
+            return format_html('<a href="' + obj_url + '">' + obj_url + "</a>")
```

Comment:
This is not a safe way of building HTML, use: https://docs.djangoproject.com/en/2.1/ref/utils/#django.utils.html.format_html